app/models/ems_refresh/refreshers/container_refresher_mixin.rb

```diff
@@ -0,0 +1,28 @@
+module EmsRefresh
+  module Refreshers
+    module ContainerRefresherMixin
```

Comment:
@moolitayer is rubocop complaining about: `module EmsRefresh::Refreshers::ContainerRefresherMixin` ?